src/main/java/bio/terra/service/filedata/flight/ingest/IngestDriverStep.java

```diff
@@ -61,6 +62,29 @@
   private final UUID profileId;
   private final CloudPlatform platform;
 
+  public IngestDriverStep(
+      LoadService loadService,
+      ConfigurationService configurationService,
+      JobService jobService,
+      String datasetId,
+      String loadTag,
+      int maxFailedFileLoads,
+      int driverWaitSeconds,
+      UUID profileId,
+      CloudPlatform platform,
+      Predicate<FlightContext> skipCondition) {
+    super(skipCondition);
+    this.loadService = loadService;
```

Comment:
It pains me that these classes don't use chained constructors. To do it you'd have to add the always false method to the base class, which wouldn't be too much work, but might not be worth doing now.